packages/client-cli/test/cli-openapi-additional-properties.test.mjs

```diff
@@ -17,5 +17,6 @@ test('export formdata on full request object', async (t) => {
   export type GetSampleResponseOK = {
     'entities': { 'foo'?: Record<string, { 'id'?: string; 'name'?: string }>; 'bar'?: Record<string, { 'type'?: 'boolean' | 'list'; 'values'?: Array<{ 'id': string; 'isArchived'?: boolean; 'isDefault': boolean; 'value': string }> }>; 'baz'?: Record<string, { 'id'?: string; 'name'?: string }> };
     'errors': { 'types'?: Record<string, { 'cause'?: unknown; 'type'?: 'notFound' | 'other' }>; 'group': Record<string, { 'cause'?: unknown; 'type'?: 'notFound' | 'other' }> };
+    'xyz'?: number;
```

Comment:
`xyz` is not itself a `number`, it's an object with numbers as values. So, this should be `Record<string, number>` or `{[key: string]: number}`.

Reply:
[f1fdc0a](https://github.com/platformatic/platformatic/pull/3405/commits/f1fdc0a817c2dc43a2995b3dd92171312ec002e2)